aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -613,14 +614,13 @@ void LazyBackendsAutogradOverridesAutogradKernel(DispatchKey key) {
   EXPECT_FALSE(called_nonautograd);
 }
 
+// no longer test ::Lazy key here
+// since it is now registered to TS backend in-tree and thus behaves differently,
+// does not throw the expected 'could not run..' messages
```

Comment:
I was trying to think of why your changes would affect these tests, since they all use a custom operator (`_test::dummy`). And... it's probably because we now register a CPU fallback kernel to the Lazy key in core? It would probably be better if we just arranged for this test to get built without any of the dispatcher registrations running :p.

Either way, removing seems totally fine.

Reply:
yes, the fallback affects these.

actually, there is another PR out (https://github.com/pytorch/pytorch/pull/73973) which I wanted you to take a look at, and I plan to land after this.  Once that's in, i think we could more easily run tests like this, however, it also doesn't seem like that important of a test to me given the other coverage we'll be adding